packages/react/src/components/Copy/Copy.js

```diff
@@ -6,91 +6,98 @@
  */
 
 import PropTypes from 'prop-types';
-import React, { Component } from 'react';
+import React, { useState, useRef, useEffect } from 'react';
 import classnames from 'classnames';
 import { settings } from 'carbon-components';
 
 const { prefix } = settings;
 
-export default class Copy extends Component {
-  static propTypes = {
-    /**
-     * Pass in content to be rendred in the underlying <button>
-     */
-    children: PropTypes.node,
+export default function Copy({
+  children,
+  className,
+  feedback,
+  feedbackTimeout,
+  onClick,
+  ...other
+}) {
+  const [animation, setAnimation] = useState('');
+  const timeoutId = useRef(undefined);
+  const classNames = classnames(className, {
+    [`${prefix}--copy-btn--animating`]: animation,
+    [`${prefix}--copy-btn--${animation}`]: animation,
+  });
+  const feedbackClassNames = classnames(
```

Comment:
Seems that we no longer need to use `classnames` here...?

Reply:
removed for `Copy` and `CopyButton`